lib/jsonapi_spec_helpers/error.rb

```diff
@@ -0,0 +1,2 @@
+class JsonapiSpecHelpersError < StandardError
```

Comment:
Could we make this

```ruby
module JsonapiSpecHelpers
  module Errors
    class Base < StandardError
    class IncludeOutOfBounds < Base
  end
end
```

To set up a generic error handling pattern for the future? This is similar to what we have in compliable.

Reply:
All set. I am pretty inexperienced with Ruby, so I am glad you directed me to a better way of dealing with errors. I will be using that pattern in my own project as needed.